src/pages/purchase-orders/edit/components/Actions.tsx

```diff
@@ -39,12 +39,14 @@ import {
 } from 'react-icons/md';
 import { useQueryClient } from 'react-query';
 import { useNavigate } from 'react-router-dom';
+import { Divider } from 'components/cards/Divider';
 
 export type Action = (po: PurchaseOrder) => {
   label: string;
   onClick: () => unknown;
   hideIf?: boolean;
   icon?: ReactElement;
+  showDivider?: boolean;
```

Comment:
Can we use the std actions array for this? Looks like we are doubling the interface.

https://github.com/invoiceninja/ui/blob/main/src/pages/quotes/common/hooks.tsx#L338

Reply:
Sure, the code has been refactored with the current convenction of handling more_actions.